lib/availability_status_listener.rb

```diff
@@ -27,6 +27,12 @@ def subscribe_to_availability_status
         process_event(event)
       end
     end
+  rescue Kafka::ConnectionError => e
```

Comment:
Regarding my fixes in the https://github.com/RedHatInsights/topological_inventory-ingress_api/pull/102 and https://github.com/RedHatInsights/topological_inventory-ingress_api/pull/103 this class will be unknown with the rdkafka solution.